crawl-ref/source/main.cc

```diff
@@ -1797,9 +1797,9 @@ void process_command(command_type cmd)
         break;
 
     case CMD_INSPECT_FLOOR:
-        request_autopickup();
         if (player_on_single_stack() && !you.running)
             pickup(true);
+        else autopickup(true);
```

Comment:
```suggestion
        else
            autopickup(true);
```

Crawl house style tweak.